src/main/scala/org/constellation/consensus/ConsensusManager.scala

```diff
@@ -51,7 +51,7 @@ class ConsensusManager[F[_]: Concurrent: ContextShift: Timer](
   nodeId: Id,
   keyPair: KeyPair,
   checkpointsQueueInstance: DataResolverCheckpointsEnqueue[F]
-) {
+)(implicit F: Concurrent[F], CS: ContextShift[F], T: Timer[F]) {
```

Comment:
you need F only explicitly

Reply:
I guess we wanted to have it uniform, so I moved all of them to this notation. I can move only what I need.